server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -1432,7 +1444,7 @@ public void verify(String seed, DiscoveryNode localNode) {
     public String toString() {
         return "BlobStoreRepository[" +
             "[" + metadata.name() +
-            "], [" + blobStore() + ']' +
+            "], [" + blobStore.get() + ']' +
```

Comment:
Not related to the other changes, but this one was weird as it created a side effect (lazy initializing the `blobStore`) in the  `toString` and I added that fix here.